ClientRuntimes/CSharp/Microsoft.Rest.ClientRuntime/ServiceClient.cs

```diff
@@ -207,10 +207,37 @@ protected void InitializeHttpClient(HttpClientHandler httpClientHandler, params
             FirstMessageHandler = currentHandler;
             HttpClient = newClient;
             Type type = this.GetType();
-            HttpClient.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue(type.FullName,
-                GetClientVersion()));
+            //setting userAgentBelow is removed because now the client can set it using SetUserAgent method
+           /* HttpClient.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue(type.FullName,
+                GetClientVersion()));*/
+        }
+        
+        //A mehtod to set user agent
```

Comment:
Typo in this comment. Please use [XML documentation comment blocks](https://msdn.microsoft.com/en-us/library/b2s063f7(v=vs.100).aspx) for documenting methods